sources/api/migration/migrator/src/main.rs

```diff
@@ -185,7 +185,11 @@ pub(crate) fn run(args: &Args) -> Result<()> {
 /// Generates a random ID, affectionately known as a 'rando', that can be used to avoid timing
 /// issues and identify unique migration attempts.
 fn rando() -> String {
-    thread_rng().sample_iter(&Alphanumeric).take(16).collect()
+    thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(16)
+        .map(char::from)
+        .collect()
```

Comment:
Would you mind testing this bit?  Independently of migrator would be fine.  There were a few changes to Alphanumeric / char handling, and I just want to be sure this still generates a rando OK.

Reply:
Is a unit test OK? What's the assertion? Just that len==16 and is_alphanumeric?

Reply:
Looks good to me:

```rust
use rand::distributions::Alphanumeric;
use rand::{thread_rng, Rng};

fn main() {
    for _ in 0..10 {
        let random_id: String = thread_rng()
            .sample_iter(&Alphanumeric)
            .take(16)
            .map(char::from)
            .collect();
        println!("{}", random_id)
    }
}
```

```text
TakPKvEYjfUTActX
fnqlC8k9vjeLXBH1
y1fHGADCkFboNh5l
eF3FqK9L0ORGIrH8
wXU5GXogiTT9W0Wd
kAVPs4GRsuDj3p4q
0WMbnQkdXgF3WNLs
HW4ciPwl11guLLn9
WxIknhrWN2MmcAMP
4iw9Y9sADf8WLxTG
```